src/main/webapp/app/text-editor/franc.ts

```diff
@@ -0,0 +1,11 @@
+export enum FrancLanguage {
```

Comment:
When I input a Spanish sentence ("Una oración alemana es una oración alemana o no?") I get English as a result.
I'm not sure if we want to handle this atm, but if we would then we have to list more languages here.

Reply:
If a student submits in a language, that isn't either German or English, it would be 0 points anyway, . I also don't think a student will submit in a "wrong" language.

Reply:
It might still be good to avoid getting English in such a case. Then we should either have the correct language or undefined!

Reply:
This was resolved, but you did not change anything. Judging from my and Stephan's feedback I would have expected that you find a way to output undefined in case of a language that is not listed (as Spanish).